src/main/scala/edu/arizona/sista/reach/extern/export/MentionManager.scala

```diff
@@ -140,6 +140,8 @@ class MentionManager {
         }
 
       case evm: EventMention =>
+        if (evm.isInstanceOf[BioEventMention])
+          mStrings += s"${indent}is-direct: ${evm.asInstanceOf[BioEventMention].isDirect}"
```

Comment:
Instead of this, maybe it would be cleaner to pattern match on `mention.toBioMention` and change each of these cases to their bio counterpart (ex. `case evm: EventMention` => `case evm: BioEventMention`).  If you match against the `Bio` form, you won't need this `isInstance` & `asInstance` stuff.